parastell/invessel_build.py

```diff
@@ -622,24 +646,24 @@ def _tag_volumes_with_materials_pydagmc(self):
 
     def generate_components_pydagmc(self):
         """Use PyDAGMC to build a DAGMC model of the invessel components"""
-        if np.isclose(
-            (self._repeat + 1) * self.radial_build.toroidal_angles[-1], 360
-        ):
-            e = AssertionError(
-                "The PyDAGMC workflow does not support modeling 360-degree "
-                "geometries. For configurations with more than one period, "
-                "please consider modeling only one period. i.e. set "
-                "'repeat = 0'."
-            )
-            self._logger.error(e.args[0])
-            raise e
         self._logger.info(
             "Generating DAGMC model of in-vessel components with PyDAGMC..."
         )
+
+        if (
+            self.radial_build.toroidal_angles[-1]
+            - self.radial_build.toroidal_angles[0]
+            == 360.0
+        ):
+            continuous_geom = True
```

Comment:
For some reason `continuous_geom` isn't an obvious name to me.  My first thought was `closed_geom` but I guess a single period with end caps is also closed... `full_360` or something like that??? I'm not feeling very creative...